### PR TITLE
Nullabel Enum bugFix

### PR DESCRIPTION
**Pull request by DamDam64**

### CODE REVIEW

**Comment by DamDam64** [comment]
1.) Example to prodcue a nullable enum Bug.
2.) Improve system to fix the bug

**Comment by DamDam64** [comment]
Reopen: I have not got any response if ....

File: Source/Examples/PropertyGrid/SimpleDemo/MainWindow.xaml.cs [hunk 1]
```diff
@@ -31,5 +32,133 @@ public MainWindow()
             InitializeComponent();
             this.DataContext = new MainWindowViewModel();
         }
+
```
Comment (objorke) [inline Source/Examples/PropertyGrid/SimpleDemo/MainWindow.xaml.cs]: the simple demo is not simple any more if we merge all these changes... ;-)

File: Source/PropertyTools.Wpf/DataGrid/Definitions/PropertyDefinition.cs [hunk 2]
```diff
@@ -299,7 +299,30 @@ public virtual FrameworkElement CreateEditControl(string bindingPath)
         /// </summary>
         protected void SetEnumItemsSource()
         {
+#if NotFixedNullableEnumBug
             this.ItemsSource = Enum.GetValues(this.PropertyType);
+#else
+            Type propType = this.PropertyType;
```
Comment (objorke) [inline Source/PropertyTools.Wpf/DataGrid/Definitions/PropertyDefinition.cs]: isn't this for the DataGrid? and the example was PropertyGrid?

**Comment by objorke** [comment]
See branch https://github.com/objorke/PropertyTools/compare/datagrid/nullable-enum and commit https://github.com/objorke/PropertyTools/commit/8ad0d6f87838d4c989d1eb268a775cc261474df0.
This is how I want the example. I have added a partial solution - the missing bit is how to set the nullable enum to null...

**Comment by objorke** [comment]
Nullable enums in the PropertyGrid should be solved in PR #130

**Pull request closed without merging**